### PR TITLE
Docker Image Context Fix

### LINKED ISSUE #11627: Bug Report: Docker Release script broken for `vtadmin`, `logrotate` and `logtail`
### Overview of the Issue

It was noticed during the v15.0.0 release, that the docker release script didn't work for `vtadmin`, `logrotate` and `logtail` images.

Errors look something like this - 
```
[+] Building 3.3s (8/9)                                                                                                                                                                                                                                                                                                                                              
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                                                                                                                            0.0s
 => => transferring dockerfile: 1.20kB                                                                                                                                                                                                                                                                                                                          0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                                                                                               0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                                                                                                 0.0s
 => [internal] load metadata for docker.io/library/debian:buster-slim                                                                                                                                                                                                                                                                                           3.2s
 => [auth] library/debian:pull token for registry-1.docker.io                                                                                                                                                                                                                                                                                                   0.0s
 => [1/4] FROM docker.io/library/debian:buster-slim@sha256:702bc27ef4be73f9c1d73c1a2bc58987c59b6fc8e8e04f46c166849817ce95dc                                                                                                                                                                                                                                     0.0s
 => [internal] load build context                                                                                                                                                                                                                                                                                                                               0.0s
 => => transferring context: 2B                                                                                                                                                                                                                                                                                                                                 0.0s
 => ERROR [2/4] COPY docker/k8s/logrotate/logrotate.conf /vt/logrotate.conf                                                                                                                                                                                                                                                                                     0.0s
 => ERROR [3/4] COPY docker/k8s/logrotate/rotate.sh /vt/rotate.sh                                                                                                                                                                                                                                                                                               0.0s
------
 > [2/4] COPY docker/k8s/logrotate/logrotate.conf /vt/logrotate.conf:
------
------
 > [3/4] COPY docker/k8s/logrotate/rotate.sh /vt/rotate.sh:
------
failed to compute cache key: failed to walk /var/lib/docker/tmp/buildkit-mount614652714/docker/k8s/logrotate: lstat /var/lib/docker/tmp/buildkit-mount614652714/docker/k8s/logrotate: no such file or directory

```

### Reproduction Steps

Try running the `release.sh` script in the docker directory.

### Binary Version

```sh
v15 and main
```


### Operating System and Environment details

```sh
all
```


### Log Fragments

_No response_

### PR DESCRIPTION
**Pull request by GuptaManan100**
## Description

During the release of v15, a failure of the docker release script was noticed. The bug outlining the issue is at https://github.com/vitessio/vitess/issues/11627. 

The issue was tracked to the PR https://github.com/vitessio/vitess/pull/10968 which changed the context that the Dockerfiles were expecting to be run from. So when the `release.sh` script is run, it fails because the file `docker/k8s/logrotate/logrotate.conf` doesn't exits. Instead it should be `logrotate.conf`.

This PR fixes this issue. It also adds the `release.sh` as a step to the release process which was previously missing.

## Related Issue(s)

- Fixes #11627 

## Checklist

-   [x] "Backport me!" label has been added if this change should be backported
-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required

### CODE REVIEW

**Comment by GuptaManan100** [comment]
I have verified manually that the script now works.

File: docker/release.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-vt_base_version='v13.0.0'
+vt_base_version='v16.0.0'
```
Comment (GuptaManan100) [inline docker/release.sh:4]: Is this step of updating the base-version something we should change in the `do-release` script automatically? cc - @frouioui

Comment (deepthi) [reply docker/release.sh:4]: that's a good idea.
Or we should include in the post-release PR where we update the versions in a bunch of files (like pom.xml).

**Review by deepthi — APPROVED** [review]

**Review by deepthi — APPROVED** [review]

**Review by rsajwani — COMMENTED** [review]
Looks good to me.

File: docker/release.sh (hunk 1 above)
Comment (GuptaManan100) [reply docker/release.sh:4]: Yep. I'll talk to @frouioui and do a follow-up PR for this change.

Comment (GuptaManan100) [reply docker/release.sh:4]: I created an issue for this at #11681 and a PR with the changes to accomplish this at #11682

**Pull request merged**